bitnami/postgresql-repmgr/10/debian-11/rootfs/opt/bitnami/scripts/librepmgr.sh

```diff
@@ -237,6 +252,60 @@ repmgr_get_primary_node() {
     echo "$primary_port"
 }
 
+
+
+########################
+# switchover standby after long primary wait
+# Globals:
+#   REPMGR_*
+#   POSTGRESQL_*
+# Arguments:
+#   None
+# Returns:
+#   None
+#########################
+
+repmgr_switchover_standby() {
+    local -i timeout="${REPMGR_WAIT_PRIMARY_TIME:-60}"
+
+    primary_host=""
+    primary_port="${REPMGR_PORT_NUMBER}"
+
+    info "Switchover from standby to primary . After unavailable primary for '$timeout' seconds . Assuming the primary role..."
+
+    echo "promote_me" >> "$REPMGR_PROMOTE_STANDBY_LOCK_FILE_NAME"
```

Comment:
You can create an empty file using `touch`:
```suggestion
    touch "$REPMGR_PROMOTE_STANDBY_LOCK_FILE_NAME"
```